src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
it was created by react-scripts automatically, when I used the command they recommend to add typescript to the project. Not sure if it's actually needed.

Reply:
Ok, then it's probably just a placeholder for adding custom type definitions in the future. Doesn't hurt to keep this (and at some point in the future it might save a few minutes googling-time ;) )